.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
 jobs:
   run_pre_commit:
     docker:
-      - image: quay.io/astronomer/ci-pre-commit:2022-04
+      - image: quay.io/astronomer/ci-pre-commit:2022-03
```

Comment:
Why was this downgraded?

Reply:
We did it because tests were failing on workers. Let me try restoring it.

Reply:
Done